.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
```

Comment:
Would you limit to DS at first?
```suggestion
    directory: "/packages/design-system"
```

Reply:
we use a repository to be aligned on all dependencies why should I limit analysis to design-system ?